### ISSUE TITLE
paralell projection?

### ISSUE THREAD

**Issue by Brian151** [kevinroast/phoria.js#33]
How might I enable parallel projection?
I would like like to be able to render things in isometric.

**Comment by kevinroast** [comment]
I haven't implement a true orthographic projection, but you can fake it easy enough by using a camera matrix with a very very long focal length and distance.

**Comment by Brian151** [comment]
In the event anyone else should come-along with same problem, I've basically figured it out...

scene perspective : 
FOV: 1

camera lookat : 
x: 0
y: 0
z: 0

camera position:
x: -800
y: 400
z: -800

That's for 2:1 isometric, anyways... (which is what I'll probably be using the most)
From what I can gather, once the first two tidbits are done, other variations of orthographic can be achieved simply by moving the camera around.

For future reference, you might want to explain things a little more clearly, as I had to do some trial and error to actually figure this out. [following] is a modified copy of the textured cube test with the GUI controls modified to allow the extremes required to make this happen. It doesn't actually set those variables, however. 

`<!DOCTYPE html>
<html>
   <head>
      <meta charset="utf-8">
      <title>Phoria - Dev test page 0</title>
      <script src="scripts/gl-matrix.js"></script>
      <script src="scripts/phoria-util.js"></script>
      <script src="scripts/phoria-entity.js"></script>
      <script src="scripts/phoria-scene.js"></script>
      <script src="scripts/phoria-renderer.js"></script>
      <script src='scripts/dat.gui.min.js'></script>
      <script>
var requestAnimFrame = window.requestAnimationFrame || window.webkitRequestAnimationFrame ||
                       window.mozRequestAnimationFrame || window.msRequestAnimationFrame || 
                       function(c) {window.setTimeout(c, 15)};
/**
   Phoria
   pho·ri·a (fôr-)
   n. The relative directions of the eyes during binocular fixation on a given object
*/

// bind to window onload event
window.addEventListener('load', onloadHandler, false);
var bitmaps = [];
function onloadHandler()
{
   // get the images loading
   var loader = new Phoria.Preloader();
   for (var i=0; i<6; i++)
   {
      bitmaps.push(new Image());
      loader.addImage(bitmaps[i], 'images/texture'+i+'.png');
   }
   loader.onLoadCallback(init);
}
function init()
{
   // get the canvas DOM element and the 2D drawing context
   var canvas = document.getElementById('canvas');

   // create the scene and setup camera, perspective and viewport
   var scene = new Phoria.Scene();
   scene.camera.position = {x:0.0, y:5.0, z:-15.0};
   scene.perspective.aspect = canvas.width / canvas.height;
   scene.viewport.width = canvas.width;
   scene.viewport.height = canvas.height;

   // create a canvas renderer
   var renderer = new Phoria.CanvasRenderer(canvas);

   // add a grid to help visualise camera position etc.
   var plane = Phoria.Util.generateTesselatedPlane(8,8,0,20);
   scene.graph.push(Phoria.Entity.create({
      points: plane.points,
      edges: plane.edges,
      polygons: plane.polygons,
      style: {
         shademode: "plain",
         drawmode: "wireframe",
         linewidth: 0.5,
         objectsortmode: "back"
      }
   }));
   var c = Phoria.Util.generateUnitCube();
   var cube = Phoria.Entity.create({
      points: c.points,
      edges: c.edges,
      polygons: c.polygons
   });
   for (var i=0; i<6; i++)
   {
      cube.textures.push(bitmaps[i]);
      cube.polygons[i].texture = i;
   }
   scene.graph.push(cube);
   scene.graph.push(Phoria.DistantLight.create({
      direction: {x:0, y:-0.5, z:1}
   }));

   var pause = false;
   var fnAnimate = function() {
      if (!pause)
      {
         // rotate local matrix of the cube
         cube.rotateY(0.5*Phoria.RADIANS);

```
     // execute the model view 3D pipeline and render the scene
     scene.modelView();
     renderer.render(scene);
  }
  requestAnimFrame(fnAnimate);
```

   };

   // add GUI controls
   var gui = new dat.GUI();
   var f = gui.addFolder('Perspective');
   f.add(scene.perspective, "fov").min(0).max(175);
   f.add(scene.perspective, "near").min(1).max(100);
   f.add(scene.perspective, "far").min(1).max(1000);
   //f.open();
   f = gui.addFolder('Camera LookAt');
   f.add(scene.camera.lookat, "x").min(-100).max(100);
   f.add(scene.camera.lookat, "y").min(-100).max(100);
   f.add(scene.camera.lookat, "z").min(-100).max(100);
   f.open();
   f = gui.addFolder('Camera Position');
   f.add(scene.camera.position, "x").min(-1000).max(1000);
   f.add(scene.camera.position, "y").min(-1000).max(1000);
   f.add(scene.camera.position, "z").min(-1000).max(1000);
   f.open();
   f = gui.addFolder('Camera Up');
   f.add(scene.camera.up, "x").min(-10).max(10).step(0.1);
   f.add(scene.camera.up, "y").min(-10).max(10).step(0.1);
   f.add(scene.camera.up, "z").min(-10).max(10).step(0.1);

   // key binding
   document.addEventListener('keydown', function(e) {
      switch (e.keyCode)
      {
         case 27:
         {
            pause = !pause;
            break;
         }
         case 32:
         {
            for (var i=0; i<6; i++)
            {
               //cube.textures.push(bitmaps[i]);
               cube.polygons[i].texture = 5-i;
            }
            break;
         }
      }
   }, false);

   // start animation
   requestAnimFrame(fnAnimate);
}
      </script>
   </head>

   <body style="background-color: #bfbfbf">
      <canvas id="canvas" width="768" height="512" style="background-color: #eee"></canvas>
      <div><a style="color:#225588;text-decoration:none;" href="./index.html">&lt;&lt; Phoria demos</a></div>
   </body>
</html>`

[attached] is what I managed to get after some messing around

![canvas](https://cloud.githubusercontent.com/assets/10995730/19066422/72664e5e-89d6-11e6-9631-17cbf4c4f457.png)

Are you still working on this library, by the way?
Just curious
